lib/modules/filteReddit.js

```diff
@@ -37,6 +37,7 @@ module.options = {
 	// valid types are: text, boolean (if boolean, value must be true or false)
 	// for example:
 	NSFWfilter: {
+		title: 'NSFW Filter',
```

Comment:
:question: [you added it](https://github.com/dhensche/Reddit-Enhancement-Suite/commit/feab9d3b9566057af13022b943da27170d5dad60#diff-05f66782ac4227bad34f2cb8c38cc6c6R171) -- so I guess it is used for the dynamically generated options for media hosts